quiz.js

```diff
@@ -0,0 +1,65 @@
+/*This is what the tree needs to look like*/
+//       *
+//      ***
+//     *****
+//    *******
+//   *********
+//  ***********
+// *************
+
+
+/*This section takes HTML fieldsets and turns them
+into variables*/
+
+var height = document.getElementById("height");
+var character = document.getElementById("character");
+var submit = document.getElementById("submit");
+function christmasTree () {
```

Comment:
This looks like it will be fun, but right now it does nothing. It should either be deleted or commented out.